build.gradle

```diff
@@ -15,10 +15,16 @@ buildscript {
 	dependencies {
 		classpath 'io.spring.gradle:spring-build-conventions:0.0.25.RELEASE'
 		classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
+		classpath 'io.spring.nohttp:nohttp-gradle:0.0.2.RELEASE'
```

Comment:
Can we add this to `spring-build-conventions` so that we only need to apply the plugin in the project build? Spring Security would benefit of this too.

Reply:
Yes. I just haven't done it yet.